station/utils/editwatcher.py

```diff
@@ -0,0 +1,24 @@
+class EditWatcher:
+    def __init__(self, obj):
```

Comment:
What does this magical class do and what is it for? Lots of python magic and not many clues... 😉

Reply:
Sure, this class is not used now. It was designed to easier edit the configuration in CLI.